docs/de_DE/README.md

```diff
@@ -11,7 +11,9 @@ sidebarDepth: 2
 
 <a :href="$themeConfig.variables.discord" target="_blank" style="float: right;"><img src="https://discord.com/api/guilds/254167535446917120/embed.png?style=banner1" alt="Unser Discord-Server: United Dispatch" data-prevent-zooming></a>
 
-[Server-Status](https://status.lss-manager.de)
+[LSSM-Server-Status](https://status.lss-manager.de)
+
+[LSS-Server-Status](https://stats.uptimerobot.com/3OVrETmBrM/786124522)
```

Comment:
use https://stats.uptimerobot.com/OEKDJSpmvK here too

Reply:
Where have i seen that link before?

Reply:
Here: https://github.com/LSS-Manager/LSSM-V.4/pull/357#discussion_r541658491

Reply:
😂 I don't know 😅

Reply:
Ah, now, thanks @JRH-1997 for sharing my monitoring page 😛